src/operator/tensor/matrix_op.cc

```diff
@@ -261,7 +317,9 @@ Example::
 .set_num_outputs(1)
 .set_attr<nnvm::FInferShape>("FInferShape", FlattenShape)
 .set_attr<nnvm::FInferType>("FInferType", ElemwiseType<1, 1>)
+#if MXNET_USE_MKLDNN == 1
```

Comment:
If MKLDNN supports this op, please add `.set_attr<bool>("TIsMKLDNN", true)` for it.

Reply:
This attribute is in Line 327.